packages/monaco-editor-wrapper/src/editorVscodeApi.ts

```diff
@@ -8,16 +8,30 @@ export type VscodeUserConfiguration = {
     json?: string;
 }
 
-export type EditorVscodeApiConfig = {
+export type EditorAppConfigVscodeApi = {
+    editorAppType: 'vscodeApi';
     extension?: IExtensionManifest | object;
     extensionFilesOrContents?: Map<string, string | URL>;
     userConfiguration?: VscodeUserConfiguration;
 }
 
-export class EditorVscodeApi extends MonacoEditorBase implements MonacoEditorWrapper {
+export class EditorAppVscodeApi extends EditorAppBase implements MonacoEditorWrapper {
```

Comment:
Would be great to add some documentation to these classes as well, even just a short description for an overview.

Reply:
Yes, good point.

Reply:
Will come